frontend/src/views/Bridge.vue

```diff
@@ -373,19 +375,19 @@ export default Vue.extend({
       storedNftsIds: [] as string[],
       currentChain: '',
       targetChain: '',
-      targetChainId: '',
+      targetChainId: 0 as number | null,
       supportedChains: [] as string[],
-      supportedChainIds: [] as string[],
+      supportedChainIds: [] as number[],
       transferStatus: '',
       transferStates,
-      currentTransferNFTId: '',
+      currentTransferNFTId: 0 as number | null,
```

Comment:
We should usually use `undefined` as a "no value", this way, in interfaces, when you declare the field that can be either a number or a "no value", you can use
`currentTransferNFTId?: number`, this way it's optional.

But it's nothing to worry about, just a tip